universal-application-tool-0.0.1/app/views/admin/ti/TrustedIntermediaryGroupListView.java

```diff
@@ -41,10 +41,10 @@ public TrustedIntermediaryGroupListView(AdminLayout layout) {
   public Content render(List<TrustedIntermediaryGroup> tis, Http.Request request) {
     ContainerTag body =
         body(
-            renderHeader("Trusted Intermediary Groups"),
-            renderTiGroupCards(tis, request),
-            renderHeader("Create New Group").withClass(Styles.MT_8),
-            renderAddNewButton(request));
+            renderHeader("Create New Trusted Intermediary").withClass(Styles.MT_8),
```

Comment:
Nit (non-blocking): consider moving this into the renderHeader function, if it can be shared between two invocation.

Reply:
nvm, looks like this is from existing code.

Reply:
@ndmckinley fyi